proposed/0021-MPID-Messaging-delete/0021-MPID-Messaging-delete.md

```diff
@@ -0,0 +1,75 @@
+- Feature Name: Deletion
+- Type: Enhancement
+- Related components: [mpid_messaging](https://github.com/maidsafe/mpid_messaging), [safe_vault](https://github.com/maidsafe/safe_vault), [safe_client](https://github.com/maidsafe/safe_client)
+- Start Date: (fill me in with today's date, DD-MM-YYYY)
+- RFC PR: (leave this empty)
+- Issue number: (leave this empty)
+
+# Summary
+
+The `MpidMessageWrapper` enum used as the serialised `PlainData::value` for MPID-messaging `Put/Post` requests/responses should for the sake of consistency include `Delete` requests. 
```

Comment:
Perhaps an overview of what the purpose of this Delete is and why it is necessary.